function-logs/handler.go

```diff
@@ -0,0 +1,99 @@
+package function
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/openfaas/openfaas-cloud/sdk"
+)
+
+// Handle grabs the logs for the fns that are named in the input
+func Handle(req []byte) string {
+
+	user := string(req)
+	var function string
+	if len(user) == 0 {
+		if query, exists := os.LookupEnv("Http_Query"); exists {
+			vals, _ := url.ParseQuery(query)
+			userQuery := vals.Get("user")
+			function = vals.Get("function")
+			if len(userQuery) > 0 {
+				user = userQuery
+			}
+		}
+	}
+
+	if len(user) == 0 {
```

Comment:
I know it doesn't make sense, but you need exit with a non-0 exit-code and to write to stderr for this to be returned as a HTTP 500 with the classic template.

This will return as a 200.